plugin/bitrise/envstore.go

```diff
@@ -0,0 +1,8 @@
+package bitrise
+
+type EnvironmentItemModel map[string]string
+
+// EnvsSerializeModel ...
+type EnvsSerializeModel struct {
```

Comment:
should this be private instead of public?  do you plan to use this data object outside the package?

Reply:
I would recommend against using `Model` in struct names.  This is a common convention in Java, but it is not really used in Go programs.

Reply:
Sure changing it.